pkg/handlers/launch.go

```diff
@@ -348,177 +312,30 @@ func (h *launchHandler) getWaitTime() int64 {
 	return 60
 }
 
-func (h *launchHandler) ServeHTTP(resp http.ResponseWriter, req *http.Request) {
-	cmdLog := createLogEntry(req)
+func (h *launchHandler) createSingleRequestHandler(resp http.ResponseWriter, req *http.Request) *singleRequestHandler {
+	return newSingleRequestHandler(resp, req, h)
+}
```

Comment:
This function seems a bit useless. You could just call `newSingleRequestHandler`

Reply:
Done, thanks :)